### ISSUE TITLE
Comments on "MRTK2 to MRTK3 - simple global speech recognition"

### ISSUE THREAD

**Issue by LocalJoost** [LocalJoost/BlogComments#426]
Original article: https://localjoost.github.io/MRTK2-to-MRTK3-simple-global-speech-recognition/

**Comment by AnhPhuN** [comment]
Hi, I was wondering if there is any extension to this article with regard to dictation in general? Transcribing speech instead of just recognizing keywords for MRTK3? Thanks.

**Comment by Jalynnn** [comment]
Thank you firstly for providing this information! However, PhraseRecognitionSubsystem no longer appears to be available. I'm not sure where this option roots from. However, I think it has been replaced with KeywordRecognitionSubsystem. This creates issues in recognizing what SpeechUtils is.

**Comment by LocalJoost** [comment]
Correct. As of the latest MRTK3 drop can no longer use 
`SpeechUtils.GetSubsystem();`

but you should use:
 `IKeywordRecognitionSubsystem phraseRecognitionSubsystem = XRSubsystemHelpers.KeywordRecognitionSubsystem;`
(or
`var phraseRecognitionSubsystem = XRSubsystemHelpers.KeywordRecognitionSubsystem;`
whatever you like)

Good luck!

**Comment by Jalynnn** [comment]
Thank you, I was able to get custom keywords to work in my VE :) As I am attaching the scripts to pressable buttons, how might you go about making these keywords require focus?

**Comment by sarellano20** [comment]
Work on oculus quest 2/pro?

**Comment by LocalJoost** [comment]
Nope, this is using Windows/Microsoft packages. AFAIK HoloLens only. Maybe Windows Mixed Reality headsets 🤪

**Comment by TimHerb2005** [comment]
hi and thank you for your post, my question is: is there any way for the keywordrecognition to hear german too and not just english ?

**Comment by TimHerb2005** [comment]
in mrtk3

**Comment by TimHerb2005** [comment]
i mean is there a way that the keywordrecognition listens for german and english phrases

**Comment by LocalJoost** [comment]
@TimHerb2005 Hello, I am not sure, maybe if you put your HoloLens in German language mode, but I never tried. If that does not work, you might try to use in phonetical English written German phrases - I have tried to do get it to recognize Dutch phrases that way. But with limited success, and your mileage may very.

**Comment by TimHerb2005** [comment]
@LocalJoost Hi Hi, It's alright, I figured it out last evening. I just had to download the German language package with speech recognition and text-to-speech in my Windows settings 😅😅.

**Comment by LocalJoost** [comment]
@TimHerb2005 Na jah das ist einfach, fabelhaft, und einfach fabelhaft ;)
